.github/workflows/rust.yml

```diff
@@ -89,4 +89,4 @@ jobs:
       - uses: actions-rs/clippy-check@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          #args: --no-deps -- -D warnings
+          args: --all-targets --all-features -- -D warnings
```

Comment:
Currently, there are a lot of warnings, some harder to fix than others. I'm not sure how beneficial this would be to have this change right now, especially considering the potential time to spend to fix it. Could it be an idea to just have it as an issue for the "real" Ecdar people for them to fix when we are finished with this semester?

Reply:
Yeah you are right. I was optimistic and thought we could maybe fix all those warnings in the branch this pull request is for.